### ISSUE TITLE
Crash with EnderIO and Thermal Expansion on Chunk Unload.

### ISSUE THREAD

**Issue by alexanderpas** [SleepyTrousers/EnderIO#2321]
java.lang.NullPointerException: Exception while updating neighbours
        at cofh.thermalexpansion.block.device.TileActivator.onRedstoneUpdate(TileActivator.java:152)
    [...]
    -- Block being updated --
    Details:
        Source block type: ID #1601 (tile.blockConduitBundle // crazypants.enderio.conduit.BlockConduitBundle)

Pastebin of full Crash Report: http://pastebin.com/C5FJSnfX

Mod Versions:

ThermalExpansion: [1.7.10]4.0.1-182
EnderIO: 1.7.10-2.2.8.367

Bug report on Thermal Expansion the side: [/CoFH/Feedback/issues/431](/CoFH/Feedback/issues/431)

**Comment by 6210classick** [comment]
what type of redstone conduit are you using ? the normal one or the insulated one

**Comment by Wetteborn** [comment]
To pipe in, I've experienced the same thing on my server (1.7.10) a few days ago with the Ender IO insulated redstone conduit that was in a bundle (redstone, item, and energized was in the bundle) connected to a Thermal Expansion, Autonomous Activator. To prevent the crash on unloading of chunk I had to connect each type from different sides of the Autonomous Activator.  Left the chunk server crashed.

Same with Advanced Genetics DNA Breeder (redstone insulated, energized and a pressurized fluid; fluid was just passing by and not connected). Left the chunk server crashed. Left unconnected to avoid crash.
